index.html

```diff
@@ -9521,6 +9524,50 @@ <h2>A project for first-time contributions</h2>
                     </ul>
                 </div>
             </div>
+            <!-- ________ Monty´s's card START ________  -->
+            <div class="card">
+                <p class="name">Monty</p>
+                <p class="contact">
+                    <i class="fab fa-twitter"></i>
+                    <a href="https://twitter.com/lmontoya" target="_blank">@lmontoya</a>
+                </p>
+                <p class="about">
+                    Content creator. Coffee lover . Collaborating from the north of Spain. 
+                </p>
+                <div class="resources">
+                    <p>3 Useful Dev Resources</p>
+                    <ul>
+                        <li>
+                            <a
+                                href="https://fakeinet.com/"
+                                target="_blank"
+                                title="Internet Fake and Scam"
+                            >
+                            My most beloved project
+                            </a>
+                        </li>
+                        <li>
+                            <a
+                                href="https://comiendoconmonty.com/"
+                                target="_blank"
+                                title="Comiendo con Monty"
+                            >
+                            Speaking of food and social media
+                            </a>
+                        </li>
+                        <li>
+                            <a
+                                href="https://wordpress.tv/"
+                                target="_blank"
+                                title="My WordPress contributions"
+                            >
+                            Papers in different WordCamps
+                            </a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+            <!-- ________ Monty´s card END ________  -->
```

Comment:
Great, you have moved the card from the top of the document. But we still have one small problem, It needs to be above the template card. 
You can see in this image, just on top of the template card there's this comment in the code, where it says "Paste YOUR CARD directly ^^ ABOVE ^^ this line". You need your card **above** it directly.
Almost there, and then I can merge it.
![image](https://user-images.githubusercontent.com/29199184/74606426-90253a80-50d0-11ea-9139-d2f41b358f97.png)